hybridgroup/commander
# Show commands from loaded command set. Display proper layout.

## How to test

1) Visit your 'command set' page, example: http://localhost:8100/#/command_set/0
2) Optional, if you don't have any command set previously saved, comment lines 1 and 48 onto the _app.js_ file.
3) You should see different layout views depending on your command set `type` params, example: 

_List layout_
http://take.ms/J5Mpr

_D-pad layout_ (pending)
http://take.ms/SHpD3
